test/lua_test.exs

```diff
@@ -76,13 +88,13 @@ defmodule LuaTest do
         """
 
       assert_raise Lua.CompilerException, error, fn ->
-        Lua.load_lua_file!(Lua.new(), path)
+        Lua.load_file!(Lua.new(), path)
```

Comment:
Add a test for loading files with the `.lua` extension. We should allow this and strip it